docs/workflows/authentication.rst

```diff
@@ -0,0 +1,75 @@
+the signature.
+
+.. code-block:: python
+
+    TOKEN_SERVER = "localhost:24816/token"
+    TOKEN_SIGNATURE_ALGORITHM = 'ES512'
+    PUBLIC_KEY_PATH = '/tmp/public.pem'
+    PRIVATE_KEY_PATH = '/tmp/private.pem'
+
+To learn more about Pulp settings, take a look at `Configuration
+<https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html>`_.
+
+Run the command ``prestart`` in order to reload the updated settings. Pulp will fetch a domain for
+the token server and initializes all handlers according to that. Test if the token authentication
+was successfully configured by initiating the following set of commands in your virtual environment::
+
+    $ http 'http://localhost:24816/v2/'
+
+    HTTP/1.1 401 Access to the requested resource is not authorized.
+    Content-Length: 93
+    Content-Type: text/plain; charset=utf-8
+    Date: Sun, 08 Sep 2019 15:29:38 GMT
+    Server: Python/3.7 aiohttp/3.5.4
+    Www-Authenticate: Bearer realm="http://localhost:24816/token",service="v2",scope="repository::pull"
+
+    401: Access to the requested resource is not authorized.
+
+Send a request to a specified realm::
+
+    $ http http 'http://localhost:24816/token?service=v2&scope=repository::pull'
+
+    HTTP/1.1 200 OK
+    Content-Length: 570
+    Content-Type: application/json; charset=utf-8
+    Date: Sun, 08 Sep 2019 15:34:41 GMT
+    Server: Python/3.7 aiohttp/3.5.4
+
+    {
+        "expires_in": 300,
+        "issued_at": "2019-09-08T15:34:41.305911Z",
+        "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6IkdGVEQ6S05CVDpIQTNUOkVOVEM6R1k0VzpHWkJWOk1FM1c6R05KVjpHSlJHOkNPREY6TVUyVzpNWVpSIn0.eyJhY2Nlc3MiOlt7InR5cGUiOiJyZXBvc2l0b3J5IiwibmFtZSI6IiIsImFjdGlvbnMiOlsicHVsbCJdfV0sImF1ZCI6InYyIiwiZXhwIjoxNTY3OTU3MTgxLCJpYXQiOjE1Njc5NTY4ODEsImlzcyI6ImxvY2FsaG9zdDoyNDgxNi90b2tlbiIsImp0aSI6ImVjYjkzN2EwLTBkYWUtNGM2My05M2YwLWZjMzE2ZDMxODI2YyIsIm5iZiI6MTU2Nzk1Njg4MSwic3ViIjoiIn0.kxS96KLLlPp9cID1SUrl9_SsXlEPdRJM5EqfwOJ1W5VIr7U2F2nJeQohaAv1IYJDmRYM1fRXPcxgBHesyIFM2Q"
+    }
+
+Use a generated token to access the root again::
+
+    $ http 'localhost:24816/v2/' --auth-type=jwt --auth="eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6IkdGVEQ6S05CVDpIQTNUOkVOVEM6R1k0VzpHWkJWOk1FM1c6R05KVjpHSlJHOkNPREY6TVUyVzpNWVpSIn0.eyJhY2Nlc3MiOlt7InR5cGUiOiJyZXBvc2l0b3J5IiwibmFtZSI6IiIsImFjdGlvbnMiOlsicHVsbCJdfV0sImF1ZCI6InYyIiwiZXhwIjoxNTY3OTU3MTgxLCJpYXQiOjE1Njc5NTY4ODEsImlzcyI6ImxvY2FsaG9zdDoyNDgxNi90b2tlbiIsImp0aSI6ImVjYjkzN2EwLTBkYWUtNGM2My05M2YwLWZjMzE2ZDMxODI2YyIsIm5iZiI6MTU2Nzk1Njg4MSwic3ViIjoiIn0.kxS96KLLlPp9cID1SUrl9_SsXlEPdRJM5EqfwOJ1W5VIr7U2F2nJeQohaAv1IYJDmRYM1fRXPcxgBHesyIFM2Q"
+
+    HTTP/1.1 200 OK
+    Content-Length: 2
+    Content-Type: application/json; charset=utf-8
+    Date: Sun, 08 Sep 2019 15:35:32 GMT
+    Server: Python/3.7 aiohttp/3.5.4
+
+    {}
+
+After performing multiple HTTP requests, the root responded with a default value ``{}``. The same
```

Comment:
Can you re-phrase, that received token can be used in accessing all the endpoints related to the requested scope ( i.e. repo) including root endpoint?